package.json

```diff
@@ -118,8 +118,7 @@
     "glob": "^7.1.3",
     "html-webpack-plugin": "^3.2.0",
     "husky": "^3.0.5",
-    "jest": "^25.1.0",
-    "jest-environment-jsdom-sixteen": "^1.0.3",
```

Comment:
Jsdom v16 is now the default in Jest, and this package doesn't work with jest v26 anyway.